Server build broken in latest 3.2 branch

**Godot version:**
`3.2` (f442dc062af3f54ef123afb4b18f4b9545728d44)


**OS/device including version:**
Linux Server (`scons p=server`)


**Issue description:**
When building for the server platform, I run into this issue:
```
Compiling ==> platform/server/os_server.cpp
In file included from ./core/cowdata.h:37,
                 from ./core/ustring.h:35,
                 from ./core/os/thread.h:35,
                 from ./core/io/resource_loader.h:34,
                 from ./drivers/dummy/texture_loader_dummy.h:34,
                 from platform/server/os_server.h:34,
                 from platform/server/os_server.cpp:31:
./drivers/dummy/rasterizer_dummy.h: In static member function 'static Rasterizer* RasterizerDummy::_create_current()':
./drivers/dummy/rasterizer_dummy.h:825:17: error: invalid new-expression of abstract class type 'RasterizerDummy'
  825 |   return memnew(RasterizerDummy);
      |                 ^~~~~~~~~~~~~~~
./core/os/memory.h:95:51: note: in definition of macro 'memnew'
   95 | #define memnew(m_class) _post_initialize(new ("") m_class)
      |                                                   ^~~~~~~
In file included from platform/server/os_server.cpp:34:
./drivers/dummy/rasterizer_dummy.h:796:7: note:   because the following virtual functions are pure within 'RasterizerDummy':
  796 | class RasterizerDummy : public Rasterizer {
      |       ^~~~~~~~~~~~~~~
In file included from platform/server/os_server.h:46,
                 from platform/server/os_server.cpp:31:
./servers/visual/rasterizer.h:1185:22: note:     'virtual const char* Rasterizer::gl_check_for_error(bool)'
 1185 |  virtual const char *gl_check_for_error(bool p_print_error = true) = 0;
      |                      ^~~~~~~~~~~~~~~~~~
```

This happens when the `Rasterizer` API has been changed without syncing the dummy implementation of `RasterizerDummy`.
This comes from c2290dbedd0, poke @lawnjelly.

I don't know why the CI builds didn't catch this.


**Steps to reproduce:**
- `scons p=server`

**Minimal reproduction project:**
n/a

--- a/drivers/dummy/rasterizer_dummy.h
+++ b/drivers/dummy/rasterizer_dummy.h
@@ -831,6 +831,8 @@ public:
 
 	virtual bool is_low_end() const { return true; }
 
+	virtual const char *gl_check_for_error(bool p_print_error = true) { return nullptr; }
+
 	RasterizerDummy() {}
 	~RasterizerDummy() {}
 };

